src/mixins/meta.ts

```diff
@@ -61,15 +80,23 @@ export function PageMetaI18nMixin(
         metaI18nTitle: string;
         metaI18nDescription: string;
         metaI18nRoutePath: string;
+        metaI18nAlternateLocale?: QMetaI18nAlternate;
       }
     ) {
+      // Parses options
+      const alternate = this.metaI18nAlternateLocale;
+      let alternateMeta = pageAlternateLocales(alternate?.locales ?? [], (options?.alternate.openGraph && alternate?.currentLocale) ? alternate.currentLocale : undefined );
+
       return {
         title: this.metaI18nTitle,
-        meta: pageSocialMetaTags(
+        meta: Object.assign(pageSocialMetaTags(
```

Comment:
Una destrutturazione con spread sarebbe più leggibile, c'è un motivo particolare per usare `Object.assign`?
Sarebbe bene usare la destrutturazione anche sopra per dare dei nomi migliori alle variabili.
E credo che vada bene usare il `const` qui, visto che non mi pare venga mai riassegnato

Es.
```ts
const { metaAlternate: pageI18nMetaTags, links: pageI18nLinks } = pageAlternateLocales(/**/)

return {
  meta: {
    ...pageSocialMetaTags(/**/),
    ...pageI18nMetaTags
  },
  links: pageI18nLinks
}
```

Reply:
In reality I just went with the fastest way I could come up at development time. But after digging a little bit inside the _Object.assign_ mechanism I have to say it is not the best choice. [Here is why](https://thecodebarbarian.com/object-assign-vs-object-spread.html) (do not mind the benchmark part, I am not sure how it was tested).
Edited to use ES6 spread syntax.